viewer/packages/cad-geometry-loaders/src/sector/culling/TakenSectorTree.ts

```diff
@@ -30,13 +30,13 @@ export class TakenSectorTree {
 
   private _totalCost: SectorCost = { downloadSize: 0, drawCalls: 0, renderCost: 0 };
 
-  constructor(sectorRoot: SectorMetadata, determineSectorCost: DetermineSectorCostDelegate) {
+  constructor(sectorRoot: BaseSectorMetadata & V8SectorMetadata, determineSectorCost: DetermineSectorCostDelegate) {
```

Comment:
Rename `TakenSectorTree` to something that includes `v8` in the name if it's specific to v8